test/e2e/app-dir/app/index.test.ts

```diff
@@ -323,24 +323,20 @@ describe('app dir - basic', () => {
     expect(res.headers.get('Content-Type')).toBe('text/x-component')
   })
 
-  it('should return the `vary` header from edge runtime', async () => {
+  it('should not return the `vary` header from edge runtime', async () => {
```

Comment:
Non-blocking but I suggest we remove these tests rather than refactoring them to explicitly state "should not", because:

- the only reason we shouldn't return them is because we decided not to, but there isn't an inherent reason why they shouldn't respond with a vary header if we decided to in the future
- the test itself isn't particularly meaningful anymore. if anything, these should be confirming that the `?_rsc` param is present on the navs, but we likely have test(s) for that elsewhere

Reply:
👍 makes sense it's less meaningful, dropped them in the new commit